libs/protos/protos/sp_external.proto

```diff
@@ -59,10 +59,14 @@ service External {
   rpc GetNotification(GetNotificationRequest) returns (GetNotificationResponse);
 
   // Attach a notification config to a pipeline
-  rpc AttachNotification(AttachNotificationRequest) returns (protos.StandardResponse);
+  rpc AttachNotification(AttachNotificationRequest) returns (protos.StandardResponse) {
+    option deprecated = true;
```

Comment:
Is this for deprecating methods? If so - can you stick a comment in there with a date noting why it was deprecated?

Reply:
Done!